restoreProvider.sh

```diff
@@ -53,10 +53,17 @@ validateClusterRequirement() {
 }
 
 checkDeployerCSV() {
+
+  csv=$(kubectl get csv | grep managed-fusion | awk '{print $1;exit}')
+  if [ -z "$csv" ];
+  then
+    csv=$(kubectl get csv | grep ocs-osd-deployer | awk '{print $1;exit}')
+  fi
+
   while true
   do
-    csvStatus=$(kubectl get csv | grep managed-fusion | awk '{ print $6; exit }')
-    echo -e "${Blue}Waiting for managed-fusion CSV to come in Succeeded phase, current csvStatus is ${EndColor}"$csvStatus
+    csvStatus=$(kubectl get csv $csv -oyaml | yq '.status .phase')
+    echo -e "${Blue}Waiting for "$csv" CSV to come in Succeeded phase, current csvStatus is ${EndColor}"$csvStatus
```

Comment:
echo -e "${Blue}Waiting for ${EndColor}"$csv"${Blue} CSV to come in Succeeded phase, current csvStatus is ${EndColor}"$csvStatus